test/end-to-end/shared/components/bhFiscalPeriodSelect.js

```diff
@@ -21,8 +21,5 @@ module.exports = {
 
     opts = bhFiscalPeriod.element(by.model('$ctrl.selectedPeriodTo'));
     opts.element(getValue(periodTo_id)).click();
-
-    // click away to free the space
```

Comment:
Did it since a long time but If my memory serves me well, the component was inside a modal so no way to click on the logo which is behind the modal.